docs/graphs/getting-started-with-graphs.md

```diff
@@ -0,0 +1,217 @@
+---
+title: Getting Started with Graphs
+sidebar_position: 10
+---
+
+In this getting started guide, you'll explore how to work with graphs in Macrometa by establishing relationships between collections, enabling you to run complex queries on airline flight data.
+
+## About Graphs
+
+Graphs provide a mechanism to model your data and perform sophisticated queries across interconnected documents. A graph is composed of _vertices_ and _edges_.
+
+- Vertices are stored in collections and linked by an edge document. Vertices can be either a document or an edge.
+- Edges are stored as documents in edge collections. The edge definition delineates which collections are employed in a named graph.
+- A named graph must contain at least one edge definition.
+
+By transforming documents into graph structures, you can conduct semantic queries using nodes, edges, and properties. Graphs directly connect data items between different collections. You can use graphs to refer to documents in different tables without a nested join. Graphs can also find patterns of document connections, such as the shortest path between two vertices in a graph.
```

Comment:
some changes to keep the terms consistent:
node -> vertex
table -> collection

"Graphs can also find patterns of document connections"
We're talking about patterns in the connectivity between documents. I think it should be "in" rather than "of".

Reply:
I read that last bit as It was finding patterns of connections. It really comes down to the meaning.
Are we:

- Using graphs to find patterns in connections, as in the pattern is in the connections?
Or
- Using graphs to find patterns of connections, as in the connections are the pattern?